xml/s4s_about.xml

```diff
@@ -210,6 +210,158 @@
   </para>
   <para condition="noquick">
    For more information about &ph;, see <xref linkend="sec-packagehub"/>.
+   For information about life cycle and support of modules and extensions, see
+   <xref linkend="sec-s4s-service"/>.
   </para>
  </sect1>
+
+ <sect1 xml:id="sec-s4s-service">
+  <title>Included Services</title>
+
+  <variablelist>
+   <varlistentry>
+    <term><emphasis>Extended Service Pack Overlap Support (ESPOS)</emphasis>
+    </term>
+    <listitem>
+     <para>
+      Subscriptions for &sles4sap; include Extended Service Pack Overlap
+      Support. It extends the overlap between the support periods of two
+      consecutive service packs by three years. During this period, you
+      receive support and all relevant maintenance updates under the terms of
+      Long Term Service Pack Support (LTSS). This feature is only available
+      with &sles4sap;.
+     </para>
+     <para>
+      Extended Service Pack Overlap Support allows you to perform service pack
+      migrations within three and a half (3½) years instead of only six
+      months. This enables you to schedule migrations more easily and perform
+      testing before a migration under less restrictive time constraints.
+     </para>
+     <para>
+      At an additional cost, &suse; also offers Long Term Service Pack Support
+      (LTSS). With LTSS, you receive support for a particular service pack
+      after the ESPOS period ends. &sles4sap; includes one and a half
+      (1½) years of general support and three years of ESPOS for each service
+      pack. Therefore, LTSS may only be offered for the last service pack. For more
```

Comment:
ESPOS is not for the final SP available.

So, proposal for rewrite (please edit!): "For each SP except the final one, SLES4SAP includes 1 1/2 years of general support followed by 3 years of ESPOS. The last SP of a release has a separate lifecyle without ESPOS as there is no overlap to the next service pack. Instead, LTSS is offerend for this last service pack.